UnboundLib/Cards/CustomCard.cs

```diff
@@ -91,7 +91,8 @@ public virtual string GetModName()
 
                 // Apply card data
                 newCardInfo.cardStats = customCard.GetStats() ?? new CardInfoStat[0];
-                newCard.gameObject.name = newCardInfo.cardName = customCard.GetTitle().Sanitize();
+                newCardInfo.cardName = customCard.GetTitle();
+                newCard.gameObject.name = $"__{customCard.GetModName()}__{customCard.GetTitle()}".Sanitize();
```

Comment:
Wouldn't it be better to just add some kind of "GetUniqueTitle" to the CustomCard, so that the resulting prefab id is not only (implicitly) available here?